packages/front-end/pages/experiments/index.tsx

```diff
@@ -414,224 +262,71 @@ const ExperimentsPage = (): React.ReactElement => {
+                    align="start"
+                    justify="between"
+                    mb="4"
+                    wrap="wrap"
+                  >
+                    <Box flexBasis="300px" flexShrink="0">
+                      <Field
+                        placeholder="Search..."
+                        type="search"
+                        {...searchInputProps}
                       />
-                    )}
-                    {canAddTemplate &&
-                    !templates.length &&
-                    allExperiments.length >= 5 ? (
-                      <div className="row justify-content-center m-3">
-                        <ExperimentTemplatePromoCard
-                          hasFeature={hasTemplatesFeature}
-                          onClick={() => setOpenTemplateModal({})}
+                    </Box>
+                    <ExperimentSearchFilters
+                      searchInputProps={searchInputProps}
+                      syntaxFilters={syntaxFilters}
+                      setSearchValue={setSearchValue}
+                      experiments={allExperiments}
+                    />
+                  </Flex>
+                  <TabsContent value="all">
+                    <ExperimentsListTable
+                      tab="all"
+                      SortableTH={SortableTH}
+                      filtered={filtered}
+                      isFiltered={isFiltered}
+                      project={project}
+                    />
+                  </TabsContent>
+                  {["running", "drafts", "stopped", "archived"].map((tab) => {
+                    if (tab === "archived" && !hasArchived) return null;
+                    return (
+                      <TabsContent value={tab} key={tab}>
+                        <ExperimentsListTable
+                          tab={tab}
+                          SortableTH={SortableTH}
+                          filtered={filtered.filter((e) => e.tab === tab)}
+                          isFiltered={isFiltered}
+                          project={project}
                         />
-                      </div>
-                    ) : null}
-                  </>
-                )
-              )}
-            </TabsContent>
-            <TabsContent value="templates">
-              <TemplatesPage
-                setOpenTemplateModal={setOpenTemplateModal}
-                setOpenDuplicateTemplateModal={setOpenDuplicateTemplateModal}
-              />
-            </TabsContent>
-          </Tabs>
+                      </TabsContent>
+                    );
+                  })}
+                  {canAddTemplate &&
+                  !templates.length &&
+                  allExperiments.length >= 5 ? (
+                    <div className="row justify-content-center m-3">
+                      <ExperimentTemplatePromoCard
+                        hasFeature={hasTemplatesFeature}
+                        onClick={() => setOpenTemplateModal({})}
```

Comment:
Do we still want it to do this? Or should this just link to the new templates page?

Reply:
Fixed.